lib/ansible/modules/packaging/os/redhat_subscription.py

```diff
@@ -129,7 +142,32 @@
     password: somepass
     autosubscribe: true
 
-# Same as above but with pulling existing system data.
```

Comment:
As per house style please use `-name:` rather than comments. If you could update the existing examples in here are the hat would be great, they must have been missed when we did the bulk tidy up. 

http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#examples-block